scripts/MPIFileHandler.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file isn't really well placed in the scripts folder. Maybe best to move it to qcore?

Reply:
Moved to score